help/destinations/catalog/advertising/tradedesk-emails.md

```diff
@@ -0,0 +1,122 @@
+---
+title: The Trade Desk - CRM
+description: Activate profiles to your Trade Desk account for audience targeting and suppression based on CRM data.
+---
+
+# The Trade Desk - CRM
+
+
+## Overview {#overview}
+
```

Comment:
I forgot to mention something here, but in the overview section we are asking partners to add a note that indicates that the documentation is partner-authored. Please edit the note below with relevant TTD information and add it to the documentation page.

```
>[!IMPORTANT]
>
>This documentation page was created by the *YOURDESTINATION* team. For any inquiries or update requests, please contact them directly at *Insert link or email address where you can be reached for updates*
```

This requirement is also in the self-service documentation template.